.github/workflows/linux_builds.yml

```
@@ -48,6 +48,16 @@ jobs:
             build-mono: false
             artifact: true
 
+          - name: Minimal Template (target=release, tools=no, everything disabled)
+            cache-name: linux-template-minimal
+            target: release
+            tools: false
+            tests: false
+            sconsflags: >
+              disable_3d=yes disable_advanced_gui=yes deprecated=no minizip=no xaudio2=no
```

```suggestion
              disable_3d=yes disable_advanced_gui=yes deprecated=no minizip=no
```
Unnecessary as it's already disabled by default